src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,18 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
 
+        Machine[] arrayMachine = new Machine[3];
+        arrayMachine[0] = new Truck();
+        arrayMachine[1] = new Bulldozer();
+        arrayMachine[2] = new Excavator();
+
+        for (Machine it : arrayMachine) {
```

Comment:
just some formatting things
```suggestion
        Machine[] machinesArray = new Machine[3];
        machinesArray[0] = new Truck();
        machinesArray[1] = new Bulldozer();
        machinesArray[2] = new Excavator();

        for (Machine machine : arrayMachine) {
```